config/stemmer.yml

```diff
@@ -100,6 +100,7 @@ protected_words:
   - ΑΦΟΙ
   - ΑΦΟΥ
   - ΑΦΟΤΟΥ
+  - ΒΙΟΛΑ
```

Comment:
I think I should I add exceptions for other cases (e.g. `βιόλας`) and/or numbers (e.g. `βιόλες`) too?